### PR TITLE
Vendor anchoring [Fix?]

### PR DESCRIPTION
**Pull request by Cdey78**
Vendors can now be anchored, WHOOP!

🆑 Cdey
add: Vendors can now be unanchored and anchored.
/🆑

 S p e e d m e r g e ?

### CODE REVIEW

File: config/example/admins.txt [hunk 1]
```diff
@@ -6,3 +6,4 @@
 ######################################################################
 
 # not_a_user - Admin
+Cdey - Host
```
Comment (Nyvrem) [inline config/example/admins.txt:9]: Why?

**Comment by Scivirino** [comment]
I just want to know why you want +Host when you've been a coder for a day

**Comment by Nyvrem** [comment]
Won't give him host unless you use the example admins text, which you don't.

File: config/example/admins.txt (hunk 1 above)
Comment (Cdey78) [reply config/example/admins.txt:9]: This does nothing, unless you revert the admin system to the legacy system.

**Comment by Cdey78** [comment]
That doesn't even give me Host, it's literally just for testing purposes

**Pull request merged**